src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -0,0 +1,16 @@
+pub struct A320Hydraulic {
+    // Until hydraulic is implemented, we'll fake it with this boolean.
+    blue_pressurised: bool,
+}
+
+impl A320Hydraulic {
+    pub fn new() -> A320Hydraulic {
+        A320Hydraulic {
+            blue_pressurised: true,
+        }
+    }
+
+    pub fn is_blue_pressurised(&self) -> bool {
+        self.blue_pressurised
+    }
+}
```

Comment:
This is implemented in newer hydraulics already with a constant threshold of minimum pressure. Is there a precise threshold required so it don't breaks stuff in emergency generator code? Right now it is pressurise if over 300 psi.

Reply:
Although the code contains emergency generator things, those are mere placeholders. Once HYD is implemented we can integrate the two and make the emergency generator actually work. So have no fear! Can't break what is already broken 😄.